packages/fxa-payments-server/src/lib/test-utils.tsx

```diff
@@ -519,12 +520,12 @@ export function getLocalizedMessage(
   msgId: string,
   args: any
 ): string {
-  const msg = bundle.getMessage(msgId);
-  if (msg === undefined) {
+  let localizedMessage = bundle.getMessage(msgId);
+  if (localizedMessage == null || localizedMessage.value === null) {
```

Comment:
Was the `==` here intentional?

Reply:
Yes, because `getMessage` can return `undefined`. Do you think that this should be updated to `localizedMessage === undefined || localizedMessage === null || localizedMessage.value === null` ?

Reply:
Ah.  Upon checking, I see that `getMessage` doesn't return a null, so we could skip the null check.

Reply:
updated